src/Microsoft.AspNetCore.Sockets/EndPointOptions.cs

```diff
@@ -1,13 +1,13 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using Microsoft.AspNetCore.Authorization;
+using System.Collections.Generic;
 
 namespace Microsoft.AspNetCore.Sockets
 {
     public class EndPointOptions<TEndPoint> where TEndPoint : EndPoint
     {
-        public AuthorizationPolicy Policy { get; set; }
+        public IList<string> Policies { get; } = new List<string>();
```

Comment:
`AuthorizationPolicyNames`. Otherwise its just a random list of strings and I have no idea what is expected there.